runtime/jcl/common/jcldefine.c

```diff
@@ -211,18 +212,25 @@ defineClassCommon(JNIEnv *env, jobject classLoaderObject,
 	}
 
 done:
-	if ((clazz == NULL) && (currentThread->currentException == NULL)) {
-		/* should not get here -- throw the default exception just in case */
-		vmFuncs->setCurrentException(currentThread, J9VMCONSTANTPOOL_JAVALANGCLASSFORMATERROR, NULL);
-	}
-			
-	result = vmFuncs->j9jni_createLocalRef(env, J9VM_J9CLASS_TO_HEAPCLASS(clazz));
+	if (clazz == NULL) {
```

Comment:
Since I've request a real change below, it would be nice to fix this to `NULL ==`, and the one below as well.

Reply:
Done.